spinalcordtoolbox/scripts/sct_deepseg.py

```diff
@@ -188,6 +188,16 @@ def get_parser(subparser_to_return=None):
         help="Contrast of the input. Specifies the contrast order of input images (e.g. `-c t1 t2`)",
         choices=('t1', 't2', 't2star'),
         metavar=Metavar.str)
+    parser_dict['totalspineseg'].add_argument(
```

Comment:
Just noticed this places the argument in the `optional arguments` argument group, which does not follow the ALL CAPS standard of the others. Something like this will resolve that:

```suggestion
    parser_dict['totalspineseg'].optional_arggroup.add_argument(
```

We might want to do the same with the `tumor_edema_cavity_t1_t2` argument too, come to think of it

Reply:
The changes I made were a little more involved (in order to access the original `input_output` and `params` groups more easily), but it really was just an indent required. :)

https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4860/commits/dbbbd87d8fe9a82946ff41c237e0098551e07906

Reply:
Oh, whoops; I usually catch that, but mixed it up. Thanks for catching that!.